### PR TITLE
fix: [SRM-11185]: icon wrapping in Healthsources table in verify step

### PR DESCRIPTION
**Pull request by vasuki-kalluri**
…ify step

### Summary

https://harness.atlassian.net/browse/SRM-11185

#### Screenshots
**BEFORE:**
image

**AFTER:**
image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by vasuki-kalluri** [comment]
retrigger titlecheck

**Comment by vasuki-kalluri** [comment]
trigger titlecheck

**Comment by vasuki-kalluri** [comment]
trigger titlecheck

**Comment by vasuki-kalluri** [comment]
trigger titlecheck

**Comment by vasuki-kalluri** [comment]
retrigger titlecheck

File: src/modules/85-cv/components/PipelineSteps/ContinousVerification/components/HealthSources/HealthSources.module.scss [hunk 1]
```diff
@@ -47,3 +47,9 @@
   user-select: none;
   cursor: not-allowed;
 }
+
+.tableType {
```
Comment (deepesh-ui) [inline src/modules/85-cv/components/PipelineSteps/ContinousVerification/components/HealthSources/HealthSources.module.scss:51]: should be healthSourceType

**Review by deepesh-ui — APPROVED** [review]

**Review by sonaligoyal-harness — APPROVED** [review]

**Comment by vasuki-kalluri** [comment]
trigger cypress

**Review by deepesh-ui — APPROVED** [review]

**Pull request merged**